.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+	"rust-analyzer.cargo.extraEnv": {
+		"RUSTUP_TOOLCHAIN": "stable"
```

Comment:
Shouldn't this be nightly?

Reply:
also, should we commit ide settings?

Reply:
Oops! The reason for this is that rust-analyzer only functions with rust stable it seems. Didn't mean to commit this.

Reply:
> @camfairchild 
> also, should we commit ide settings?

Considering the `.vscode/tasks.json` seems to have a `tasks` list that may allow for executing code, I'd say tracking that IDE configuration to be risky, and as for other IDE configuration files I've usually read others frowning on the practice... though I have been known to use Vim comments within individual files :shrug:

Reply:
Removed files from tracking and added the directory to gitignore